src/sentry/api/serializers/models/group.py

```diff
@@ -195,8 +197,12 @@ def serialize(self, obj, attrs, user):
             status_details['autoResolved'] = True
         if status == GroupStatus.RESOLVED:
             status_label = 'resolved'
-            if attrs['pending_resolution']:
-                status_details['inRelease'] = attrs['pending_resolution']
+            if attrs['resolution']:
+                res_type, res_version = attrs['resolution']
+                if res_type in (GroupResolution.Type.in_next_release, None):
```

Comment:
I dont like the idea of trading trivial amounts of space for confusion. I intend to backfill the data and stop supporting None.